manage_arkime/core/capacity_planning.py

```diff
@@ -95,22 +96,31 @@ def to_dict(self) -> Dict[str, any]:
             "minCount": self.minCount,
         }
 
-def get_capture_node_capacity_plan(expected_traffic: float) -> CaptureNodesPlan:
+def get_capture_node_capacity_plan(expected_traffic: float, azs: List[str]) -> CaptureNodesPlan:
     """
     Creates a capacity plan for the indicated traffic load.
+
     expected_traffic: The expected traffic volume for the Arkime cluster, in Gigabits Per Second (Gbps)
+    azs: The AZs to deploy the cluster into
     """
 
     if not expected_traffic or expected_traffic < MINIMUM_TRAFFIC:
         expected_traffic = MINIMUM_TRAFFIC
 
     if expected_traffic > MAX_TRAFFIC:
         raise TooMuchTraffic(expected_traffic)
+    
+    chosen_instance = (
```

Comment:
I would prefer this continue to use the array so we can add more instances in the future. Is there a reason not to? For example there has already been a request for the free teir.

Reply:
> I would prefer this continue to use the array so we can add more instances in the future. Is there a reason not to? For example there has already been a request for the free teir.

It felt odd to me that the instance type itself would know what it's own max utilization is, and indeed, when I tried to make that amount dynamic the scheme fell apart because we only know the information we need to make that decision in this method.  This seems like the right place to make that decision from both a theoretical and practical perspective.

Reply:
Ok, well it needs to use an array and not have hard coded logic. However you want to do that.

Reply:
Not sure I'm following; don't see what's special about an array vs. other solutions?  Also, there's currently no need for an array in this case anyways and we can easily update the behavior later if/when we need to.

Reply:
So the reason that it was hard coded maxTraffic before is that was used for a lab setting. I'm going to be adding another one for a even smaller instance type. I personally really dislike global variables that have been added that are specific to a instance type that now require renaming AND changes if we want to change them.

Reply:
Just posted an update to that section of code, hopefully it satisfies your concerns?